### PR TITLE
Remove cgi dependency from Client

### PR DESCRIPTION
**Pull request by mdujava**

### CODE REVIEW

**Review by p3ck — COMMENTED** [review]
I think we will need the following to make it happy on both python2 and python3

```
# Fix Python 2.x.
try:
    UNICODE_EXISTS = bool(type(unicode))
except NameError:
    unicode = lambda s: str(s)
```

**Comment by mdujava** [comment]
Problem is when defining subclass like `class SensitiveUnicode(unicode):`

But this is not problem with the removal of the cgi dependency. Not sure why MR introducing github action did pass.

**Comment by StykMartin** [comment]
```
But this is not problem with the removal of the cgi dependency. Not sure why MR introducing github action did pass.
```
This is the first reference to the Python 3 codebase (common->client). All other uses of bkr.common.helpers were in labcontroller and server. This is why we never caught this problem.

File: Common/bkr/common/helpers.py [hunk 1]
```diff
@@ -100,7 +101,8 @@ def run(self):
             self.finished.clear()
 
 
-class SensitiveUnicode(unicode):
+@python_2_unicode_compatible
+class SensitiveUnicode():
```
Comment (JohnVillalovos) [inline Common/bkr/common/helpers.py:105]: Should this be derived from `str` ?

```suggestion
class SensitiveUnicode(str):
```

Comment (JohnVillalovos) [reply Common/bkr/common/helpers.py:105]: Actually that doesn't change the error ☹️ So never mind.

Comment (mdujava) [reply Common/bkr/common/helpers.py]: Did not find a way how to make it compatible, so I put it into separate file for now.

**Comment by mdujava** [comment]
!rebuild

File: Client/src/bkr/client/commands/cmd_update_inventory.py [hunk 2]
```diff
@@ -87,14 +87,14 @@
...
 
-import cgi
 import sys
 from xml.dom.minidom import parseString
 
 from requests.exceptions import HTTPError
 
 from bkr.client import BeakerCommand
 from bkr.client.task_watcher import watch_tasks
+from bkr.common.helpers_py3 import parse_content_type
```
Comment (StykMartin) [inline Client/src/bkr/client/commands/cmd_update_inventory.py:97]: `helpers_py3` will be used in python2 and python3 imports (as we are still building the client for Python 2). This is a bit misleading. I prefer to rename this file.

Comment (mdujava) [reply Client/src/bkr/client/commands/cmd_update_inventory.py]: Renamed it to `helpers_six` as it works with python2 and python3, when whole codebase will be python3 compatible and we drop python2 support, we could move everything to original file.

Comment (StykMartin) [reply Client/src/bkr/client/commands/cmd_update_inventory.py]: Sounds good. Thanks!

**Review by StykMartin — APPROVED** [review]

**Pull request merged**